src-vue/src/components/PlayButton.vue

```diff
@@ -39,6 +49,28 @@ export default defineComponent({
                     value: Object.keys(ReleaseCanal)[Object.values(ReleaseCanal).indexOf(v)]
                 }
             });
+        },
+        selectOptions(): {label: string, options: {value: ReleaseCanal, label: string}[]}[] {
+            return [
+                {
+                    label: 'In development',
```

Comment:
I'm considering adding a _pull-launcher-and-mods-from-main_ in the future which is closer to "In development". So maybe we should call this something that indicates that it's a release candidate that has been tested to some minimum degree. Not sure how obvious "Release Candidate"  is to players... Maybe "Beta"?

It should still indicate though that it's not considered "stable".

Reply:
Fixed in 66795215af6abd275a926859e7702224b32ff925